packages/atomic/src/components/common/facets/category-facet/search-value.spec.ts

```diff
@@ -0,0 +1,183 @@
+    props: Partial<CategoryFacetSearchValueProps> = {}
+  ) => {
+    const defaultProps: CategoryFacetSearchValueProps = {
+      value: {
+        count: 42,
+        path: ['electronics', 'computers'],
+        displayValue: 'laptops',
+      },
+      i18n,
+      field: 'category',
+      onClick: vi.fn(),
+      searchQuery: 'laptop',
+    };
+    const mergedProps = {...defaultProps, ...props};
+    const container = await renderFunctionFixture(
+      html`${renderCategoryFacetSearchValue({props: mergedProps})}`
+    );
+
+    return {
+      container,
+      li: container.querySelector('li'),
+      button: container.querySelector('button'),
+      labelSpan: container.querySelector('span[part="value-label"]'),
+      countSpan: container.querySelector('.value-count'),
+      countSpanByPart: container.querySelector('[part="value-count"]'),
+      pathContainer: container.querySelector('[part="search-result-path"]'),
+      separators: container.querySelectorAll('.mx-0\\.5'),
+      truncateSpans: container.querySelectorAll('.truncate, .max-w-max'),
+    };
+  };
+
+  it('should render a list item element', async () => {
+    const {li} = await renderComponent();
+
+    expect(li).toBeInTheDocument();
+  });
+
+  it('should render button with correct structure', async () => {
+    const {button} = await renderComponent();
+
+    expect(button).toBeInTheDocument();
+  });
+
+  it('should apply search-result part to button', async () => {
+    const {button} = await renderComponent();
+
+    expect(button).toHaveAttribute('part', 'search-result');
+  });
+
+  it('should render facet value label highlight', async () => {
+    const {labelSpan} = await renderComponent();
+
+    expect(labelSpan).toBeInTheDocument();
+  });
+
+  it('should render count with correct formatting', async () => {
+    const {countSpan} = await renderComponent();
+
+    expect(countSpan).toHaveTextContent('(42)');
+  });
+
+  it('should apply value-count part to count span', async () => {
+    const {countSpanByPart} = await renderComponent();
+
+    expect(countSpanByPart).toBeInTheDocument();
+  });
+
+  it('should render path with "in" label', async () => {
+    const {pathContainer} = await renderComponent();
+
+    expect(pathContainer).toHaveTextContent('in');
+  });
+
+  it('should render path with category values', async () => {
+    const {pathContainer} = await renderComponent();
+
+    expect(pathContainer).toHaveTextContent('category: electronics');
+  });
+
+  it('should render path separator', async () => {
+    const {separators} = await renderComponent();
+
+    expect(separators.length).toBeGreaterThan(0);
+  });
+
+  it('should call onClick when button is clicked', async () => {
+    const onClickMock = vi.fn();
+    const {button} = await renderComponent({onClick: onClickMock});
+
+    button?.click();
+    expect(onClickMock).toHaveBeenCalled();
+  });
+
+  it('should generate correct aria-label', async () => {
```

Comment:
We're not testing the value of the label at all here. I think we should: there's quite a lot of logic involved.